modules/apps/frontend-data-set/frontend-data-set-sample-web/src/main/java/com/liferay/frontend/data/set/sample/web/internal/view/CustomizedTableFDSView.java

```diff
@@ -60,9 +69,34 @@ public FDSTableSchema getFDSTableSchema(Locale locale) {
 		).add(
 			"color", "color",
 			fdsTableSchemaField -> {
-				String moduleName = _npmResolver.resolveModuleName(
-					"@liferay/frontend-data-set-sample-web/js" +
-						"/GreenCheckDataRenderer");
+				List<FDSCellRendererCET> fdsCellRendererCETs =
+					(List)_cetManager.getCETs(
+						CompanyThreadLocal.getCompanyId(), null,
+						ClientExtensionEntryConstants.TYPE_FDS_CELL_RENDERER,
+						Pagination.of(QueryUtil.ALL_POS, QueryUtil.ALL_POS),
+						null);
```

Comment:
This seems very low-level, for this scope of code, where API is used by devs and customers. Should this be behind a more user-friendly utility?

Reply:
Yeah, we may change that when we create the UIs for mapping FDS cell renderers to FDS views. Apparently the current APIs in CETManager have been enough for everybody, but I doubt they are really using them, because they are so limited :shrug: .